src/screens/SignUpScreen/components/SignUpForm/components/FormField/index.js

```diff
@@ -1,16 +1,16 @@
 import React from "react";
 import Option from "./components/Option";
-import { ErrorMessageText } from "./styles";
+import { ErrorMessageText, InputClass } from "./styles";
 import { Field, ErrorMessage } from "formik";
 
 const FormField = ({ label, name, type, note, options }) => (
-  <div className="form-group">
-    <label>{label}</label>
-    {type !== "checkbox" && type !== "radio" && (
-      <Field name={name} type={type} className="form-control" required />
-    )}
+  <div>
+    <h4>{label}</h4>
+      {type !== "checkbox" && type !== "radio" && (
+        <Field name={name} type={type} style={InputClass} required />
```

Comment:
also small side note, I think it's convention fe react if your variable isn't a component it starts with a lowercase letter